src/components/facets/FacetMoreRowsConnected.tsx

```diff
@@ -0,0 +1,36 @@
+import { IReactVaporState, ReduxUtils, IReduxAction } from '../../utils/ReduxUtils';
+import {
+  IFacetMoreRowsOwnProps, FacetMoreRows, IFacetMoreRowsProps, IFacetMoreRowsStateProps,
```

Comment:
Usually, when imports are breaking into more that one line, it's one line per import. Simply run the "optimise import" reformatting tool in WebStorm and it should do the trick.